shared/chat/inbox/container.js

```diff
@@ -179,18 +191,36 @@ const mapStateToProps = (state: TypedState, {isActiveRoute, smallTeamsExpanded})
   }
 }
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({
+const mapDispatchToProps = (dispatch: Dispatch, {focusFilter}) => ({
   loadInbox: () => dispatch(loadInbox()),
+  onHotkey: cmd => {
+    if (cmd.endsWith('+n')) {
+      dispatch(newChat([]))
+    } else {
+      focusFilter()
+    }
+  },
   onNewChat: () => dispatch(newChat([])),
+  onSelect: (conversationIDKey: ?Constants.ConversationIDKey) =>
+    conversationIDKey && dispatch(selectConversation(conversationIDKey, true)),
   onSetFilter: (filter: string) => dispatch(setInboxFilter(filter)),
   onUntrustedInboxVisible: (converationIDKey, rowsVisible) =>
     dispatch(untrustedInboxVisible(converationIDKey, rowsVisible)),
 })
 
+const findNextConvo = (rows: I.List<any>, selected, direction) => {
```

Comment:
Not sure if this is on purpose, but if you're on the first item and hit 'up', it cycles to the last item (but not vice versa).